vendor/src/github.com/docker/libnetwork/ipams/remote/remote_regular.go

```diff
@@ -0,0 +1,28 @@
+// +build !experimental
```

Comment:
Seems almost identical to `remote_experimental.go` but pluginstore.
It would be great if this code clone is reduced